scripts/deploy.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Following assumptions are made by this script:                               #
+# * gcloud, docker, and docker-compose is installed                            #
+# * it is called from $GOPATH/src/github.com/google/keytransparency/scripts    #
+# * there is a project called key-transparency on gce which has a cluster      #
+#   called "ci-cluster" within the "us-central1-a" compute zone                #
+# * there is a service key to authenticate with glcoud to above project in     #
+#   $GOPATH/src/github.com/google/keytransparency/service_key.json             #
+################################################################################
+
+PROJECT_NAME=key-transparency
+NAME_SPACE=default
+
+MAX_RETRY=30
+
+function main()
+{
+  # create key-pairs:
+  ./prepare_server.sh -f
+  initGcloud
+  buildDockerImgs
+  tearDown
+  pushTrillianImgs
+
+  # Deploy all trillian related services:
+  kubectl apply -f ../kubernetes/trillian-deployment.yml
+
+  pushKTImgs
+  waitForTrillian
+  createTreeAndSetIDs
+
+  # Deploy all keytransparency related services (server and signer):
+  kubectl apply -f ../kubernetes/keytransparency-deployment.yml
+}
+
+function initGcloud()
```

Comment:
Try doing this in before_deploy:
https://medium.com/google-cloud/continuous-delivery-in-a-microservice-infrastructure-with-google-container-engine-docker-and-fb9772e81da7